### ISSUE TITLE
virus?

### ISSUE THREAD

**Issue by mohamadbagherblack** [1440-2b2t/Pyro-Client#16]
i dont know much about virus but i sacaned this file in https://www.virustotal.com and it detected virus with 4 anti virus.
i dont want say ur project was virus and i installed it. i just want be sure what it is.
website find these
Java.Trojan.GenericGBA.29345
Trojan.Java.GenericGBA
Artemis
im telling again i dont know much about stuff like this i just want u tell me

**Comment by dotztv** [comment]
i hvae used it and had no problems even later on, its doesnt have viruses and it is not harmful

**Comment by aeoaeoaeo** [comment]
Yes, there is some kind of virus/suspicious line of code. 1440 will claim that the obfuscation will cause false positives but obfuscation does not affect whether or not it comes up as a virus.  Here is a version that I cleaned 
https://github.com/aeoaeoaeo/Clean-Pyro-Crack

**Comment by Euro-pol** [comment]
virustotal has a lot of false positives with java code

**Comment by aeoaeoaeo** [comment]
> virustotal has a lot of false positives with java code

no, this does have something in it idk what but some sort of malware. If something has a lot of positives for malware its most likely malware

**Comment by MostlyGhostlyGaming** [comment]
jar files often get flagged as a virus and virus total isnt the best when it comes to java
